### PR TITLE
Add prettier + husky + format npm script

### PR DESCRIPTION
**Pull request by alejandronanez**
- Formatted the existing code
- Added a .prettierrc file to configure Prettier
- Add a pre-commit hook to run prettier in staged files

### CODE REVIEW

**Review by stacietaylorcima — APPROVED** [review]
So fresh and so clean. Perfect! Thanks, @alejandronanez! 👏 ❤️

File: README.md [hunk 1]
```diff
@@ -5,9 +5,16 @@ The goal of this project is to create a “smart” shopping list app that learn
 How it works is that you will enter items (e.g., “Greek yogurt” or “Paper towels”) into your list. Each time you buy the item, you mark it as purchased in the list. Over time, the app comes to understand the intervals at which you buy different items. If an item is likely to be due to be bought soon, it rises to the top of the shopping list.
 
 The app will work in many of the same ways as [iNeedToBuy.xyz](https://app.ineedtobuy.xyz/) (on which our project is based) with the exception that we will not be implementing barcode scanning (that feature would add a lot of scope to the project and in my experience wasn’t all that useful).
+
 <hr>
 
+### Code formatting
+
+We're using [Prettier](https://prettier.io/) to format our code before every commit, this is done automatically and will not affect the way you work!
+If you're curious, here's the PR that adds this feature https://github.com/the-collab-lab/smart-shopping-list/pull/30
```
Comment (segdeha) [inline README.md:14]: To match the rest of the doc, please swap `'` for `’` on this line and the previous line. Thanks!

Comment (segdeha) [reply README.md:14]: Oh whoops, merged already

**Review by segdeha — CHANGES_REQUESTED** [review]
Wanna use actual apostrophes. Thanks for humoring me!

**Pull request merged**